www/tailwind.config.js

```diff
@@ -109,13 +109,13 @@ module.exports = {
       }),
     },
   },
-  variants: {
-    extend: {
-      inset: ['group-hover'],
-      stroke: ['dark'],
-      height: ['hover'],
-    },
-  },
+  // variants: {
```

Comment:
@florian-lefebvre is this breaking something?

Reply:
I commented those lines but they could even be deleted according to [the official docs](https://tailwindcss.com/docs/just-in-time-mode#variants-are-rendered-together) since we're using the JIT mode. After clicking on the link, scroll down and you'll see ''Variant configuration (including order) is not respected by the JIT engine".